source/extensions/filters/http/transformation/transformer.h

```diff
@@ -53,32 +53,80 @@ class Transformer {
 
 typedef std::shared_ptr<const Transformer> TransformerConstSharedPtr;
 
+class RequestTransformer : public Transformer {
+public:
+  virtual ~RequestTransformer() {}
+
+  virtual bool passthrough_body() const PURE;
+
+  virtual void transform(Http::RequestOrResponseHeaderMap &map,
+                         // request header map. this has the request header map
+                         // even when transforming responses.
```

Comment:
i don't think this is accurate anymore? that was the whole point of creating separate classes?

Reply:
good call, removed the comments where no longer relevant [8e3a137](https://github.com/solo-io/envoy-gloo/pull/204/commits/8e3a137861f451dea9a888c00cc9f9e2712b8a14)